project/Dependencies.scala

```diff
@@ -98,7 +98,7 @@ object Dependencies {
 
   val workbenchModelV  = s"0.15-${workbenchLibsHash}"
   val workbenchGoogleV = s"0.21-${workbenchLibsHash}"
-  val workbenchNotificationsV = s"0.3-${workbenchLibsHash}"
+  val workbenchNotificationsV = s"0.3-3086a4df-SNAP"
```

Comment:
Will update to a non-SNAP version once https://github.com/broadinstitute/workbench-libs/pull/1054 is merged